bodhi/server/models.py

```diff
@@ -2659,9 +2659,13 @@ def build_label(build):
                 title += " more"
 
                 return title
+            elif len(self.builds) == 0:
```

Comment:
Since when there is no builds the return value is the same of beautify or not beautify, we could check if `self.builds == 0` right before the `if beautify` and return the alias, that would let us avoid duplicating that code.

Reply:
Not exactly: if `self.display_name != ''` it will take precedence over `self.builds == 0`, but only when `beautify`... thus I couldn't put the `if` before checking `beautify` value.

Reply:
ah yeah that let's packager being able to put a custom name to the update.